docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -920,80 +921,91 @@ environments use this environment as a base.
 
 **cfgrib_env**
 
-* Python 3.10.4
-* metpy 1.4.0
-* netcdf4 1.5.8
-* cfgrib 0.9.10.1
-* pygrib 2.1.4
+* Python 3.12.0
+* metpy 1.6.3
+* netcdf4 1.7.2
+* cfgrib 0.9.15
+* pygrib 2.1.6
 
 **cycloneplotter_env**
 
-* Python 3.10.4
-* cartopy 0.20.3
-* matplotlib 3.5.2
-* pandas 1.4.3
+* Python 3.12.0
+* cartopy 0.24.0
+* matplotlib 3.10.0
+* pandas 2.2.3
 
 **geovista_env**
 
-* geovista
-* xarray 2022.11.0
-* iris 3.3.1
+* Python 3.12.0
+* geovista 0.5.3
+* xarray 2025.1.2
+* iris 3.11.1
 
 **h5py_env**
 
 * All packages in py_embed_base_env
-* h5py 3.6.0
+* h5py 3.12.1
 
 **icecover_env**
 
 * All packages in py_embed_base_env
-* xarray 2022.3.0
-* pyresample 1.24.1
-* scikit-learn 1.1.1
-* pyproj 3.3.1
+* xarray 2025.1.2
+* pyresample 1.32.0
+* scikit-learn 1.6.1
+* pyproj 3.7.0
 
 **metdataio_env**
 
-* Python 3.10.4
-* lxml 4.9.1
-* pymysql 1.0.2
-* pandas 1.5.1
+* Python 3.12.0
+* pymysql 1.1.1
+* pyyaml 6.0.2
+* xarray 2025.1.2
+* lxml 5.3.0
+* netcdf4 1.7.2
```

Comment:
need to add pandas 2.2.3 (for the reformatter)

Reply:
So the environment does have pandas 2.2.3, but I didn't explicitly install it in the script that creates the conda environment. These lists correspond to the install commands in those scripts (in internal/scripts/docker_env/scripts). I didn't list the packages that are dependencies of other packages are aren't explicitly installed.